x-pack/plugins/security_solution/public/detection_engine/rule_management_ui/components/rules_table/rules_table/rules_table_saved_state.ts

```diff
@@ -10,7 +10,7 @@ import { enumeration } from '@kbn/securitysolution-io-ts-types';
 import { SortingOptions, PaginationOptions } from '../../../../rule_management/logic';
 
 export enum RuleSource {
-  Prebuilt = 'prebuilt',
+  Elastic = 'elastic',
```

Comment:
Why renamed? "Prebuilt" and "custom" rules are the two domain terms we use in our language.

Reply:
Good point, rolled back